packages/cli-hydrogen/src/cli/services/deploy/config.integration.test.ts

```diff
@@ -1,44 +1,46 @@
 import {getDeployConfig} from './config.js'
-import {describe, it, expect} from 'vitest'
-import {file, git} from '@shopify/cli-kit'
+import {gitInit} from '../../prompts/git-init.js'
+import {describe, beforeEach, it, expect, vi} from 'vitest'
+import {error, file, git} from '@shopify/cli-kit'
 
 const defaultConfig = {deploymentToken: 'abcdefg', dmsAddress: 'https://integration.test', healthCheck: true}
+const mockedGitInitPrompt = vi.fn()
+
+beforeEach(() => {
+  vi.mock('../../prompts/git-init.js')
+  vi.mocked(gitInit).mockImplementation(mockedGitInitPrompt)
```

Comment:
Even though this is an integration test we still need to mock at least the user prompt interaction.